### PR TITLE
autotest step input support point param

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
autotest step input support point param

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=227839&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiOTIiLCIxMDAxMjA1Il19&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=506&type=BUG)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  autotest step input support point param            |
| 🇨🇳 中文    |   支持自动化测试输入点           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chengjoey** [comment]
image
image
image

**Comment by sfwn** [comment]
Dot only allowed in query param key.

File: modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/helper.go [hunk 1]
```diff
@@ -169,8 +169,8 @@ const props2 string = `}
   						"msg": "参数名最大长度不能超过50"
 						},
             {
-              "pattern": "/^[a-zA-Z0-9_-]*$/",
-              "msg": "参数名为英文、数字、中划线或下划线"
+              "pattern": "/^[.a-zA-Z0-9_-]*$/",
+              "msg": "参数名为英文、数字、点、中划线或下划线"
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/helper.go:173]: Why two changes?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/helper.go (hunk 1 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/helper.go:173]: http header should use -

**Review by sfwn — APPROVED** [review]

**Pull request merged**